README.md

```diff
@@ -9,32 +9,61 @@
 Please make sure you have [**Pieces OS**](https://docs.pieces.app/installation-getting-started/what-am-i-installing) installed to run the Package.
 
 
-## Getting Started
+## Getting Started with the Pieces Sublime Plugin
```

Comment:
Hey @Bishoy-at-pieces so keeping the readme streamlined like the other readmes for our plugins, 

You can start the readme this way: 
```
# [Pieces for Developers](https://pieces.app) | Sublime Plugin

Then the hero image 

## Overview 
What is the Sublime plugin

## Features

<!--   Add all of the features here --> 
```